addons/crm/tests/test_crm_lead.py

```diff
@@ -302,6 +302,48 @@ def test_crm_lead_lost_date_closed(self):
         lead.action_set_lost()
         self.assertEqual(lead.date_closed, datetime.now(), "Closed date is updated after marking lead as lost")
 
+    @users('user_sales_manager')
```

Comment:
Side note: there is a "test_crm_lead_notification.py" file holding mail-related tests for lead,n maybe it could be moved in that file ?